dot_lpdev_cmds.sh

```diff
@@ -245,8 +245,11 @@ function __lpdev_geth_run {
 
   echo "Running Geth miner with the following command:
   geth -networkid 54321 \\
-       -rpc \\
+       -rpc -ws \\
+       -rpcaddr "0.0.0.0" -wsaddr "0.0.0.0" \\
+       -rpccorsdomain "*" -wsorigins "*" \\
        -rpcapi 'personal,account,eth,web3,net' \\
+       -wsapi 'personal,account,eth,web3,net' \\
```

Comment:
you haven't added these parameters to the command that actually runs `geth`

Reply:
Ah yep, fixed